app/scripts/actions/config.js

```diff
@@ -0,0 +1,28 @@
+import {action} from './utils'
+
+export const CONFIG = {
+  LOAD_CONFIG_PAGE: 'LOAD_CONFIG_PAGE',
```

Comment:
for the load part please use [`createPage`](https://github.com/ipfs/webui/blob/redux/app/scripts/actions/utils.js#L33)